buffer/src/main/java/io/netty/buffer/ByteBufInputStream.java

```diff
@@ -204,7 +204,8 @@ public boolean readBoolean() throws IOException {
 
     @Override
     public byte readByte() throws IOException {
-        if (!buffer.isReadable()) {
+        int available = available();
+        if (available == 0) {
```

Comment:
nit: why separate var?

Reply:
just because it was like this in other methods as well. So I felt it would be better to keep it consistent